docs/config_reference.rst

```diff
@@ -456,6 +456,20 @@ System Partition Configuration
       No other test would be able to proceed.
 
 
+.. py:attribute:: systems.partitions.sched_options.sacct_retries
+
+   :required: No
+   :default: ``3``
+
+   Some times the ``sacct`` command may be temporarily unavailable.
+   In this case, ReFrame will continue polling until ``sacct_retries`` is reached.
+   Every time the command succeeds, the counter is reset.
```

Comment:
When documenting configuration options, always put a separate single-line paragraph at the beginning as a short description. For example, here:

> Maximum number of consecutive failures of `sacct` before ReFrame reports a failure.

And thenn describe the rationale.